pom.xml

```diff
@@ -148,6 +148,22 @@
     <!-- nexus-staging-maven-plugin -->
     <autoReleaseAfterClose>true</autoReleaseAfterClose>
 
+    <!--
+      srcdeps-maven-plugin
+      let's have the SCM URLs as propeties so that they can be switched
+      to local repos on command line when testing
+    -->
```

Comment:
There already is an example in agent: https://github.com/ppalaga/hawkular-agent/commit/81f66c2867bce5f3f70a04324bc2259fd5aebcdb#diff-600376dffeb79835ede4a0b285078036R54